app/html/xcreate_user.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
```

Comment:
what's the different between create_user.html and xcreate_user.html?

Reply:
oh I should have deleted xcreate_user.html